WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1972,6 +1970,10 @@ private void saveResult(boolean saved, boolean discardable, boolean savedLocally
     }
 
     private void publishPost() {
+        publishPost(false);
+    }
+
+    private void publishPost(final boolean isDraftToPublish) {
```

Comment:
👋 @mzorz , I wonder if we can do the `PostStatus.fromPost(mPost) == PostStatus.DRAFT` check inside the method and we go back to only have one parameter-less `publish()` method.

It looks like the only time we call this method directly with a true/false literal is [from showPublishConfirmationOrUpdateIfNotLocalDraft](https://github.com/wordpress-mobile/WordPress-Android/blob/e06af40f34a4772f7376baae5b1f6f87f5360e3d/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L1495) and it seems we can afford to redo the `PostStatus.fromPost(mPost) == PostStatus.DRAFT` in that case? WDYT?

Reply:
Thank you! Let's adress separately as agreed